### DISCUSSION TITLE
Host on Custom Domain

### DISCUSSION THREAD

**Discussion by HakanOZDABAK** [deta/space-docs#50] [Q&A]
I deployed my project and I have a own domain. When I set up my project to my domain on computer and when enter my domain from my phone, my domain wants me sign in to Deta Space. How can I set on this project to online at my Custom Domain?

**Comment by sudo-subg** [comment]
hey @HakanOZDABAK, every app on Space is by default secured with authentication. When logged into Space, only the app instance owner can access it. For more information, have a look at [this](https://deta.space/migration/learn-more/auth-comparison).
You can use [public_routes](https://deta.space/docs/en/reference/spacefile#public_routes) in Spacefile, to specify which routes of your app you want to make publicly accessible without requiring Space authentication. As a hint, to make your entire app publicly accessible, you can use
```yaml
public_routes:
  - "/*"
```
